charts/cluster-autoscaler/templates/servicemonitor.yaml

```diff
@@ -20,6 +20,10 @@ spec:
   - port: {{ .Values.service.portName }}
     interval: {{ .Values.serviceMonitor.interval }}
     path: {{ .Values.serviceMonitor.path }}
+    {{- if .Values.serviceMonitor.metricRelabelings }}
+    metricRelabelings:
+{{ tpl (toYaml .Values.serviceMonitor.metricRelabelings | indent 6) . }}
```

Comment:
Please add this new field to the [values.yml](https://github.com/kubernetes/autoscaler/blob/035e762e89dade935f7954e64d78ef54512946cf/charts/cluster-autoscaler/values.yaml#L331)

Reply:
fixed